libs/wallet/src/provider.tsx

```diff
@@ -104,7 +104,7 @@ export const VegaWalletProvider = ({ children }: VegaWalletProviderProps) => {
       disconnect,
       connector: connector.current,
       sendTx,
-    };
+    } as VegaWalletContextShape;
```

Comment:
question (blocking): Why the type cast here?

Reply:
formatting error

Reply:
Hm there's a larger problem then. I have a feeling its due to sendTx types?